pkg/awsManager/ebsManager.go

```diff
@@ -40,40 +41,39 @@ func ListEbsSnapshotForDeletion(client clientpkg.Client) []*string {
 	return ebsSnapshotsToBeDeleted
 }
 
-// this deletes the Ebs Snapshot
+// DeleteEbsSnapshots deletes the Ebs Snapshot
 // successful execution returns nil. Unsuccessful execution or errors occured, would return an error
-func DeleteEbsSnapshots(client clientpkg.Client, ebsSnapshotsToBeDeleted []*string) error {
+func DeleteEbsSnapshots(client clientpkg.Client, ebsSnapshotsToBeDeleted []*string, logger logr.Logger) error {
 
 	if ebsSnapshotsToBeDeleted == nil {
 		return nil
 	}
 	var ebsSnapshotsNotDeleted []*string
-	for _, ebsSnapshotId := range ebsSnapshotsToBeDeleted {
+	for _, ebsSnapshotID := range ebsSnapshotsToBeDeleted {
 
-		_, ebsSnapshotDeleteError := client.DeleteSnapshot(&ec2.DeleteSnapshotInput{SnapshotId: ebsSnapshotId})
+		_, ebsSnapshotDeleteError := client.DeleteSnapshot(&ec2.DeleteSnapshotInput{SnapshotId: ebsSnapshotID})
 		if ebsSnapshotDeleteError != nil {
-			fmt.Println(ebsSnapshotDeleteError)
-			fmt.Print("Could not delete snapshot :", *ebsSnapshotId)
-			ebsSnapshotsNotDeleted = append(ebsSnapshotsNotDeleted, ebsSnapshotId)
+			logger.Error(ebsSnapshotDeleteError, "Failed to delete snapshot", *ebsSnapshotID)
+			ebsSnapshotsNotDeleted = append(ebsSnapshotsNotDeleted, ebsSnapshotID)
 		}
 	}
 
 	if ebsSnapshotsNotDeleted != nil {
-		return errors.New("Could not delete all snapshots for this region")
+		return errors.New("FailedComprehensiveSnapshotDeletion")
 	}
 
 	return nil
 }
 
-func ListVolumeForDeletion(client clientpkg.Client) []*string {
+func listVolumeForDeletion(client clientpkg.Client, logger logr.Logger) []*string {
```

Comment:
Why was this changed to unexported?

Reply:
because it is a helper function of `CleanEbsVolumes()`

Reply:
It has uses cases outside of CleanEbsVolumes(), and all the other AWS resource types also had them exported (some still do too)

Reply:
Maybe I am misunderstanding something. The methods I unexported were all helper methods not used anywhere else and that shouldn't be used for anything other than their `Clean` parent function

Reply:
But why shouldn't they ever? I know they aren't at the moment but I don't think that's a good reason to say they shouldn't